README.md

```diff
@@ -27,7 +27,7 @@ git clone git@github.com:springload/eslint-config-springload.git
 cd eslint-config-springload
 nvm install
 # Then, install all project dependencies.
-npm install
+yarn install
```

Comment:
`install` is redundant but sure :)

Reply:
I still use `yarn install` when in the terminal lol

Reply:
🙈